modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-pgt-image-registry_{context}"]
+= Configuring the image registry using PolicyGenTemplate CRs
+
+You can use `PolicyGenTemplate` to apply to create the PV and PVC and patch `imageregistry` configuration. Select the appropriate `PolicyGenTemplate` for each `source-cr`. See Additional Resources for more help.
+
+.Prerequisites
```

Comment:
Generally....the pre req is that the user has ZTP setup

Reply:
Added "* You have installed and configured Zero Touch Provisioning (ZTP)."